myriad-scheduler/src/main/java/org/apache/myriad/MyriadModule.java

```diff
@@ -103,7 +103,7 @@ protected void configure() {
 
     MapBinder<String, TaskFactory> mapBinder = MapBinder.newMapBinder(binder(), String.class, TaskFactory.class);
     mapBinder.addBinding(NodeManagerConfiguration.NM_TASK_PREFIX).to(NMTaskFactoryImpl.class).in(Scopes.SINGLETON);
-    Map<String, ServiceConfiguration> auxServicesConfigs = cfg.getServiceConfigurations();
+    Map<String, ServiceConfiguration> auxServicesConfigs = cfg.getServiceConfigurations().orNull();
     if (auxServicesConfigs != null) {
```

Comment:
Yah, missed that, will update

On Wed, May 11, 2016 at 10:38 PM, Darin <notifications@github.com> wrote:

> In myriad-scheduler/src/main/java/org/apache/myriad/MyriadModule.java
> <https://github.com/apache/incubator-myriad/pull/71#discussion_r62957707>:
>
> > @@ -103,7 +103,7 @@ protected void configure() {
> >
> >      MapBinder<String, TaskFactory> mapBinder = MapBinder.newMapBinder(binder(), String.class, TaskFactory.class);
> >      mapBinder.addBinding(NodeManagerConfiguration.NM_TASK_PREFIX).to(NMTaskFactoryImpl.class).in(Scopes.SINGLETON);
> > -    Map<String, ServiceConfiguration> auxServicesConfigs = cfg.getServiceConfigurations();
> > +    Map<String, ServiceConfiguration> auxServicesConfigs = cfg.getServiceConfigurations().orNull();
> >      if (auxServicesConfigs != null) {
>
> Seems like if we've got an Optional auxServices.isPresent would be
> appropriate to avoid null check.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-myriad/pull/71/files/9bb1e6b7514d58b5cb128bd3e23bec0a03f55aaa#r62957707>
>